src/js/control.js

```diff
@@ -0,0 +1,40 @@
+export class Control {
+  constructor(details, container) {
+    this.id = details.id;
+    this.title = details.title;
+    this.description = details.description;
+    this.showSwitcher = details.showSwitcher;
+    this.container = container;
+    this.element;
+
+    // Rendering off the bat here as this is a dumb component.
+    // It saves creating a variable and calling .render() on it.
+    this.render();
+  }
+
+  render() {
+    let control = document.createElement('div');
```

Comment:
This doesn't get redefined so it can be a `const`.
```suggestion
    const control = document.createElement('div');
```